ui/app/styles/core/buttons.scss

```diff
@@ -46,12 +93,8 @@ $button-box-shadow-standard: 0 3px 1px 0 rgba($black, 0.12);
     min-width: auto;
     padding: 0;
   }
-  &.is-flat {
-    min-width: auto;
-    border: none;
-    box-shadow: none;
-  }
 
```

Comment:
For context, this is the only is-danger button—found on the replication dashboard to demote a cluster. ![image](https://user-images.githubusercontent.com/6618863/227680366-3b9c0612-79fb-43bd-83b7-959341c0fb67.png)